Curve-finance-suite/Address-Provider-Ownership-Transfer/src/agent.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Finding,
+  FindingSeverity,
+  FindingType,
+  HandleTransaction,
+  ethers,
+} from "forta-agent";
+import {
+  FindingGenerator,
+  provideEventCheckerHandler,
+} from "forta-agent-tools";
+
+export const ADDRESS = "0x0000000022d53366457f9d5e68ec105046fc4383"; // Address Provider contract address
+const abi = ["event CommitNewAdmin(uint256,address)"];
+export const iface = new ethers.utils.Interface(abi); 
+
+const createFindingGenerator = (alertID: string): FindingGenerator => {
+  return (metadata: { [key: string]: any } | undefined) => {
+    return Finding.fromObject({
+      name: "Curve Admin Event Detected",
+      description: "An ownership transfer has been initiated",
+      alertId: alertID,
+      severity: FindingSeverity.Medium,
+      type: FindingType.Info,
+      metadata: {
+        newAdmin: metadata!.topics[2],
+      },
+    });
+  };
+};
+
+export const provideCommitNewAdminEvent = (
+  alertID: string,
+  address: string
+): HandleTransaction => provideEventCheckerHandler(
+      createFindingGenerator(alertID),
+      iface.getEvent('CommitNewAdmin').format('sighash'),
+      address
+    );
+
+export default {
+  handleTransaction: provideCommitNewAdminEvent(
+    "CURVE-3",
```

Comment:
Can we change this to `CURVE-1` for following the order from the table at `https://www.notion.so/nethermind/CurveDAO-7807d9f852ac42f2973c5aa4747a80af`

Reply:
We are following the order of the **Agent** section instead of table

Reply:
Ohh I see, but looks like not all of the agents are there right? The table has much more

Reply:
Yes, once you pick an agent, you need to write your agent in the list, pick the next available Id and also wrote the proposed solution.